GOVERNANCE.md

```diff
@@ -90,22 +89,60 @@ in terms of project direction, to potentially less experienced contributors.
 
 Council Members will have the responsibility of
 * Removing members, including Council Members, if they are in violation of the Code of Conduct
-* Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+* Make decisions when regular community discussion does not produce consensus on an issue
+  in a reasonable time frame.
 * Make decisions about strategic collaborations with other organizations or individuals.
 * Make decisions about the overall scope, vision and direction of the project.
 * Developing funding sources
 * Deciding how to disburse funds with consultation from Core Contributors
 
 The council may choose to delegate these responsibilities to sub-committees. If so, Council members must update this document to make the delegation clear.
 
-Note that individual council member does not have the power to unilaterally wield these responsibilities, but the council as a whole must jointly make these decisions. In other words, Council Members are first and foremost Core Contributors, but only when needed they can collectively make decisions for the health of the project.
-
-ArviZ will be holding its first election to determine its initial council in the coming weeks and this document will be updated.
+Note that individual council member does not have the power to unilaterally wield these
+responsibilities, but the council as a whole must jointly make these decisions.
+In other words, Council Members are first and foremost Core Contributors, but only
+when needed they can collectively make decisions for the health of the project.
+
+### Council Decision Making Process
+By and large we expect the decisions in ArviZ to be made _ad hoc_ and require little formal
+coordination and with the community at large. However for controversial proposals and
+new core contributors the council may need to intervene to make the final decision
+in a group vote.
+
+#### Call for a vote
+Core contributors can call for a vote on to resolve a target issue they feel has
+been stale for too long and informal consensus appears unlikely. For a voting
+to be called, the target issue must be at least 2 months old.
+
+To do so, they have to open a proposal issue ticket labeled "Council Vote".
+The proposal issue should contain a link to the target issue and a proposal
+on how to resolve it. Proposals should include a statement making clear what it
+means to "agree" or to "disagree".
+
+Before voting starts, at least 3 days will be left for Core
+Contributors to raise doubts about proposal _phrasing_, no extra discussion will take
+place in the proposal issue. Proposal issues should be locked from creation
```

Comment:
Discussion _should have already taken place_ in the target (original) issue and Slack.

GitHub issues can be locked so that only people with merge privileges can comment. I locked #1260 for example to prevent stray comments. As all core contributors will have merge privileges, this effectively allows only core members to comment with phrasing complains and council members to vote. This is to prevent comments on the issue that are not part of the voting process. These should go in the target issue which would remain unlocked.

Reply:
Ok got it, the current wording is clear then 👍